generator/handlebars/kss_handlebars_generator.js

```diff
@@ -302,13 +306,15 @@ kssHandlebarsGenerator.generatePage = function(styleguide, sections, root, secti
   }
   // Create the HTML to load the optional CSS and JS.
   for (key in this.config.css) {
-    if (this.config.css.hasOwnProperty(key)) {
-      styles = styles + '<link rel="stylesheet" href="' + this.config.css[key] + '">\n';
+    if (typeof this.config.destination !== 'undefined') {
+      cssPath = path.relative(this.config.destination, this.config.css[key]);
+      styles = styles + '<link rel="stylesheet" href="' + cssPath + '">\n';
     }
   }
   for (key in this.config.js) {
-    if (this.config.js.hasOwnProperty(key)) {
-      scripts = scripts + '<script src="' + this.config.js[key] + '"></script>\n';
+    if (typeof this.config.destination !== 'undefined') {
+      jsPath = path.relative(this.config.destination, this.config.js[key]);
+      scripts = scripts + '<script src="' + jsPath + '"></script>\n';
```

Comment:
The `--js` option takes URLs, not paths relative to the current directory. You are free to pass paths relative to the destination directory in the `--js` option, since relative paths are the same as relative URLs in that case. But you cannot change the code to no longer accept full URLs like https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js